packages/stream/src/index.ts

```diff
@@ -134,7 +168,7 @@ export const createAmplitudeStream = (
 
   let source: MediaStreamAudioSourceNode;
   createEffect(() => {
-    const currentStream = stream();
+    const currentStream = stream instanceof MediaStream ? stream : stream ? stream() : undefined;
```

Comment:
I'd rather test if typeof stream is function, makes it more legible.

Reply:
Made a new commit with that change.